client.js

```diff
@@ -196,6 +197,7 @@ LimitdClient.prototype._connectUsingReconnect = function (done) {
                     return;
                   }
                   this.emit('error', err);
+                  socket.emit('error', err);
```

Comment:
this is an error emitted by reconnect itself, I think we should not propagate it back to the socket.